sources/settings-migrations/v1.23.0/nvidia-container-runtime-settings/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "nvidia-container-runtime-settings"
+version = "0.1.0"
+authors = ["Monirul Islam <monirulu@amazon.com>"]
```

Comment:
slight nit, should @arnaldo2792 be a listed author as well? (and for other crates added in this PR)

Reply:
I wanted to keep the ownership of the work since the commits will have my name. My changes to that code were minimal :+1: